scripts/apps/search/controllers/MultiActionBarController.tsx

```diff
@@ -188,7 +192,16 @@ export function MultiActionBarController(
     };
 
     this.canPublishItem = function() {
-        return multi.getItems().every((item) => item.state !== 'draft' && $location.path() !== '/workspace/personal');
+        return multi.getItems().every((item) => {
+            if (item.state !== 'draft') {
+                if (personalLocationPath) {
+                    return appConfig.features.hasOwnProperty('publishFromPersonal')
```

Comment:
Just for checking if the property exists or not but I think this: `appConfig.features && appConfig.features.publishFromPersonal != null && appConfig.features.publishFromPersonal` is the correct format.

Reply:
Since you care about the value, the fact that it exists doesn't help much. If `appConfig.features` was null, `hasOwnProperty` call would throw an error, and if it was an object, you could as well read the value of `appConfig.features.publishFromPersonal`, even if `publishFromPersonal` isn't defined.

> appConfig.features && appConfig.features.publishFromPersonal != null && appConfig.features.publishFromPersonal

You can also use the short forrmat: `appConfig?.features?.publishFromPersonal === true`